rca/static_src/sass/components/_toggle-switch.scss

```diff
@@ -102,6 +106,15 @@
         }
     }
 
+    .programmes-categories & {
```

Comment:
It seems like the styles for this one is specific to programmes-categories so I've implemented it like this.